profile/profile.yaml

```diff
@@ -0,0 +1,93 @@
+name: "Brejvinder Singh Dhillon"
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_BrejvinderDhillon
+
+summary: I'm Brejvinder, a recent graduate from the University of Windsor with a Bachelor's in Computer Science Honors. I specialize in the design and analysis of algorithms and have a foundation in theoretical computer science. I'm currently a Data Engineer at Jarvis Consulting where I'm being given the opportunity to apply my knowledge and hone my skills with industry-standard tools. I have an interest in artificial intelligence, data analytics and game theory. Despite being from a science background, I have a passion for creative arts, namely music and creative writing which translates back into my programing as I like to create beautiful and efficient code.
+
+skills:
+  proficient:
+    - Java
+    - C/C++
+    - Algorithms/Data Structures
+    - Linux/Bash
+    - RDBMS/SQL
+    - Agile/Scrum
+    - Git/GitHub
+    - GitFlow
+    - Maven
+  competent:
+    - CSS
+    - PHP
+    - HTML
+    - JavaScript
+    - Python
+    - Docker
+    - Springboot
+    - REST APIs
+  familiar:
+    - Operating Systems
+    - Computer Graphics
+    - TeX
+    - x86 ASM
+    - Google Cloud Platform
+
+jarvis_projects:
+  - name: "Cluster Monitor"
+    git_url: "/linux_sql"
+    description: Developed a tool that allows users to monitor nodes connected in a Linux cluster by tracking the hardware specifications and resource usage of the cluster nodes. The data is collected in real-time and is currently being used for report generation and future planning. The Monitoring Agents on the nodes collect and send data to the centralized database on a scheduled basis via crontab over the network. The docker initialization and SQL scripts ensure the deployment of the RDBMS with a standardized schema and environment.
+  - name: "Core Java Apps"
+    git_url: "/core_java"
+    description:
+      - "Twitter App: Implemented an application that can Post, Show and Delete Tweets from the command line. It follows the MVC design pattern, utilizing an HTTP client library to access the Twitter REST API."
+      - "JDBC App: Coded an implementation of CRUD (Create, Read, Update and Delete) operations on a PostgresSQL Database through the Data Access Object(DAO) design pattern."
+      - "Grep App: Programmed an implementation of the Linux grep command. Specifically, it recursively searches through a specified directory for a given RegEx pattern and stores the found lines into an output file. There are two implementations, one with and without the usage of Lambda Functions."
+  - name: "Springboot App"
+    git_url: "/springboot"
+    description: "Developed an online stock trading simulation RESTful API that can be consumed by front-end and mobile developers, as well as traders. It retrieves stock market information and persists it into a database that can then be used to CRUD quote, trader and order data from and to the database."
```

Comment:
too short. you should mention the technologies used in this project, such as Spring Boot, maven, docker, testing tools, swagger, JSON etc..